src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,252 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Random;
+import java.io.FileWriter;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.IOException;
+import java.io.File;
+import duke.task.Task;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+public class Duke {
+
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static final String FILE_SEPARATOR = "CHOPCHOP";
+
+    private static void printSeparator() {
+        for(int i = 0; i < 60; i++) {
+            System.out.print('-');
+        }
+        System.out.print('\n');
+    }
+
+    private static void exitMethod() {
+        System.out.print("My cover's blown!\n");
+        printSeparator();
+    }
+
+    /**
+     * Echoes the user input with random upper and lower case for mockery.
+     *
+     * @param line User input.
+     */
+    private static void mockEcho(String line) {
+        Random rd = new Random();
+        for(int i = 0; i < line.length(); i++) {
+            if(rd.nextBoolean()) {
```

Comment:
You could leave a space before `(` as in line 51 ,to maintain layout consistency.